cmd/stakercli/daemon/daemoncommands.go

```diff
@@ -408,21 +384,22 @@ func stakeFromPhase1TxBTC(ctx *cli.Context) error {
 		blockHeighTxInclusion = uint64(resp.Blk.Height)
 	}
 
-	paramsForHeight := globalParams.GetVersionedGlobalParamsByHeight(blockHeighTxInclusion)
-	if paramsForHeight == nil {
-		return fmt.Errorf("error getting param version from global params %s with height %d", inputGlobalParamsFilePath, blockHeighTxInclusion)
+	respParamsByHeight, err := client.BtcStakingParamByBtcHeight(sctx, uint32(blockHeighTxInclusion))
+	if err != nil {
+		return fmt.Errorf("failed to get btc staking parameters: %w", err)
 	}
+	btcStakingParams := respParamsByHeight.StakingParams
 
 	stakerAddress := ctx.String(stakerAddressFlag)
-	_, err = client.BtcDelegationFromBtcStakingTx(sctx, stakerAddress, stakingTransactionHash, paramsForHeight)
+	_, err = client.BtcDelegationFromBtcStakingTx(sctx, stakerAddress, stakingTransactionHash, []byte{}, btcStakingParams.CovenantPks, btcStakingParams.CovenantQuorum)
```

Comment:
maybe we can remove ` []byte{}` from here as we always passing empy ?

In general I think it makes sense to ignore the tag here, if other data from op_return is valid i.e even if transaction had wrong tag and wouldn't be discovered in phase-1, if it is valid phase-2 tx it should be able to register,

Reply:
done at https://github.com/babylonlabs-io/btc-staker/pull/151/commits/dd5a04bb5063de1a73bc3692e9b9d772526494a7